packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Utilities/Utilities" />
+
+# Utilities
+
+## Box VS Text
+
+In production docs, these sections will live with their appropriate utility. I've co-located them here, so that we can align internally.
+
+### Box
+
+`<Box />` acts as the root for most components. Prefer spreading omitted rest props and forwarding a ref to `<Box />` as well.
+
+This component uses an `as="abbr"` polymorphic API to keep the DOM a11y compliant as well as clean and comprehensible by developers and their automated tests.
+
+`<Box />` contains a minimal set of styles that are provided through inheritance to children (`font-family`) and provides an API for utility tailwind-esque styling (`padding`, `margin`).
+
+These styles that are owned by `<Box />` are also a centralized place to apply theme-able odyssey design tokens for override (theming `font-family` "globally").
+
+Going further, `<Box />` accepts a `className` prop for applying component-specific styling. This prevents the need for `<Box />` to define every style for all use cases. Additionally, parent component classes can contain their own theme-able tokens to keep things customizable.
+
+A simple component like `<Button>` might look like this:
+
+```html
+<Box {...omitProps} ref="{ref}" as="button" className="{componentClass}">
+  { label }
+</Box>
+```
+
+**WIP API NOTE:** Can Box handle spreading props and the ref under the hood, so component authors wouldn't need to specify?
+
```

Comment:
the ref has to be provided explicitly from the top level component to the inner component (Box). That is something that has to be wired up each time as it represents an API design decision. Box itself can always forward its ref to the top most polymorphic root which I think is what you might be asking.

As for props similar answer, component authors need to decide which props provided to the parent are passed where. If Box is at the root, then 9/10 the props should be omitted and spread there. Box itself can do the same thing internal to its polymorphic root. This is good context to add to the notes above in fact

Reply:
Thanks for the clarity. I don't think this needs to be docc'd rn, but I wanted to ask the question in situ for my own understanding.

Reply:
Resolving: undone.